tests/unit/utils/test_utils_record.py

```diff
@@ -56,7 +56,7 @@ def test_get_abstract_returns_empty_string_when_no_titles():
 
 
 def test_get_title_returns_empty_string_when_titles_is_empty():
-    empty_titles = Record({'titles': []})
+    empty_titles = Record({'foo': 'bar'})
```

Comment:
This makes the test name wrong, which means that you changed the behaviour of `get_title` when a record has a `titles` key with value `[]`. If you think the test makes no longer sense, delete it, rather than leaving it with a broken name.

In fact, `get_title({'titles': []})` now returns `[]`, which is not even a string and definitely not a sensible value for a title, which means that my smart `get_value(record, 'titles[0].subtitle', default='')` wasn't so smart after all.